src/components/views/elements/AppTile.js

```diff
@@ -86,8 +90,17 @@ export default React.createClass({
         });
     },
 
-    _onEditClick: function() {
-        console.log("Edit widget %s", this.props.id);
+    _onEditClick: function(e) {
+        console.log("Edit widget ID ", this.props.id);
+        if (e) {
```

Comment:
I don't think there is any default tbh - I was just doing what the `AppsDrawer` does for `onClickAddWidget`. Will remove.